src/main/config.js

```diff
@@ -35,3 +35,7 @@ export const VIEW_MENU_ITEM = {
   'Typewriter Mode': false,
   'Focus Mode': false
 }
+
+export const LINE_ENDING_REG = /(\r\n|\n)/g
+export const LF_LINE_ENDING_REG = /([^\r]\n)/
```

Comment:
If a file only have one line, and only line is `\n`?

is the line break is LF?

So is this RegExp expression better.

```javascript
export const LF_LINE_ENDING_REG = /(^\n|[^\r]\n)/
```

But I am not test the regexp bellow.

Reply:
Good point, I changed it to `([^\r]\n)|(^\n$)`, so we can explicit match `\n`.